image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -48,18 +48,18 @@ spec:
                     values:
                     - "true"
             {{- if ._rox.env.openshift }}
-            - weight: 25
+            - weight: 50
               preference:
                 matchExpressions:
-                  - key: node-role.kubernetes.io/compute
+                  - key: node-role.kubernetes.io/infra
```

Comment:
Does the order here matter? If not then I'm not sure it's worth moving it because the highest weight rule (100) is at the bottom.

Reply:
The order is by preferred note and not by weight, the master affinity has a `NotIn` operator

Reply:
Order in the yaml doesn't matter but I think it is nice that it reflects the priority.
Affinity logic was a bit confusion for me initially. Both `infra` and `compute` has `operator: In`. It means that we add its weight to the final score if such label is presented in the node. `Infra` has higher weight so that means that it is more preferable. `Master` has max weight but it also has `operator: NotIn` so it is sort of anti-affinity.  Basically it means that if we choose between master node and something else, we always choose non master node. That's why it is in the end of the list in the yaml